packages/tools/nodeEditor/src/components/preview/previewMeshControlComponent.tsx

```diff
@@ -51,14 +63,14 @@ export class PreviewMeshControlComponent extends React.Component<IPreviewMeshCon
     }
 
     useCustomMesh(evt: any) {
-        const files: File[] = evt.target.files;
+        const files: File[] = evt.target?.files || evt.dataTransfer?.files;
         if (files && files.length) {
             const file = files[0];

```

Comment:
just wondering if this is still correct (just in case you don't want to use the files input). is the model file always the first one?

Reply:
(sorry, actually meant two lines down)

Reply:
PreviewFile isn't used by the mesh loading anymore, but it's used by the particle system so that's why I left it there. In the case of the mesh loading, it's going to always load all files.